packages/react-core/src/components/Wizard/WizardNav.tsx

```diff
@@ -26,7 +29,11 @@ export const WizardNav: React.FunctionComponent<WizardNavProps> = ({
   }
 
   return (
-    <nav className={css(styles.wizardNav, isOpen && 'pf-m-expanded')} aria-label={ariaLabel}>
+    <nav
+      className={css(styles.wizardNav, isOpen && 'pf-m-expanded')}
```

Comment:
is the class available in the styles object?

Reply:
I wasn't previously, but is now! Updated